de.bforartists.Bforartists.json

```diff
@@ -0,0 +1,178 @@
+{
+    "id": "de.bforartists.Bforartists",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "bforartists",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home",
+        "--filesystem=/media",
+        "--filesystem=/run/media",
+        "--env=TMP_DIR=/tmp",
+        "--env=TMP=/tmp"
+    ],
+    "add-extensions": {
+        "org.blender.Blender.Codecs": {
```

Comment:
Same here. You don't control this extension, and it makes no ABI stability guarantees for anything but Blender. Should be dropped.